operators/nfs-provisioner-operator/ci.yaml

```diff
@@ -0,0 +1,7 @@
+---
+# Use `replaces-mode` or `semver-mode`.
+updateGraph: replaces-mode
+
+---
```

Comment:
Please remove this line

Reply:
I removed the line

Reply:
Sorry I thought you want me to remove updateGraph param too. 
I will add updateGraph param.